Lib/test/test_syntax.py

```diff
@@ -1700,20 +1686,6 @@
    SyntaxError: multiple exception types must be parenthesized
```

Comment:
Should we call out it's only when `as` is used?

```suggestion
   SyntaxError: multiple exception types must be parenthesized when using 'as'
```